content/docs/rosa/alb-nlb-ossm-e2e/index.md

```diff
@@ -0,0 +1,530 @@
+
+![architecture diagram showing privatelink with TGW](./images/rosa_alb_tgw_nlb_e2e.png)
+
+### Prerequisites
+
+- A privatelink ROSA cluster
+- [ROSA CLI](https://github.com/openshift/rosa) - Download the latest release
+- oc CLI `rosa download openshift-client`
+- [jq](https://jqlang.github.io/jq/download/)
+
+Clone the repository
+
+```bash
+git clone https://github.com/houshym/rosa-ossm-e2e-encryption
+```
+ ### Deploy OpenShift Service Mesh(OSSM)
+
+ **Note:** if service mesh is already installed skip this step
+
+Installing the OSSM(OpenShift Service Mesh) involves installing the OpenShift Elasticsearch, Jaeger, Kiali, and Service Mesh Operators, creating and managing a ServiceMeshControlPlane resource to deploy the control plane, and creating a ServiceMeshMemberRoll resource to specify the namespaces associated with the Service Mesh 
+
+Install service mesh operators by applying the following snippet on the cluster or use this [script](./ossm-operator/deploy-ossm.sh)
+
+```bash
+cat << EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: elasticsearch-operator
+  namespace: openshift-operators
+spec:
+  channel: "stable"
+  name: elasticsearch-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic
+---
+#install jaeger operator
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: jaeger-product
+  namespace: openshift-operators
+spec:
+  name: jaeger-product
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  channel: "stable"
+  installPlanApproval: Automatic
+--- 
+#intsall Kiali operator
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: kiali-ossm
+  namespace: openshift-operators
+spec:
+  name: kiali-ossm
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  channel: "stable"
+  installPlanApproval: Automatic
+--- 
+# Create a Subscription  to subscribe the openshift-operators
+# namespace to the Red Hat OpenShift Service Mesh Operator
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: servicemeshoperator
+  namespace: openshift-operators
+spec:
+  channel: "stable"
+  name: servicemeshoperator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic
+EOF
+```
+
+Install service mesh operators by applying the following snippet on the cluster or use this [script](./ossm-operator/deploy-ossm.sh) and Check the operator's status in each cluster with the following commands or run this [script](./ossm-operator/check-ossm.sh). if you need troubleshooting follow the [troubleshooting operator](https://docs.openshift.com/container-platform/4.12/support/troubleshooting/troubleshooting-operator-issues.html) : 
```

Comment:
against scripts are not links, they're paths to the scripts in the cloned repo.

Reply:
done